src/Tasks/Common/Resources/xlf/Strings.pt-BR.xlf

```diff
@@ -1,80 +1,80 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
   <file datatype="xml" source-language="en" target-language="pt-BR" original="../Strings.resx">
     <body>
       <trans-unit id="AddResourceWithNonIntegerResource">
         <source>NETSDK1076: AddResource can only be used with integer resource types.</source>
-        <target state="translated">NETSDK1076: AddResource só pode ser usado com tipos de recursos inteiros.</target>
+        <target state="new">NETSDK1076: AddResource can only be used with integer resource types.</target>
```

Comment:
Something went wrong and things are resetting to English in some files. Can you try reverting all the xlf and let it regenerate? Also, if you have any insight on how this happened, let's add an issue to github.com/dotnet/xliff-tasks.

Reply:
Nice catch as I missed this (unexpanded) file.  I wonder what caused this.

Reply:
It happened to `src/Tasks/Common/Resources/xlf/Strings.tr.xlf` too.

Reply:
Looking through the history, this happened in the very first commit. I'll redo the work and look through each file this time... and see if I can repro the problem along the way..

Reply:
Fixed these.